### PR TITLE
Release 0.3.1

### PR DESCRIPTION
**Pull request by SpheMakh**
- Add Cubical
- Add CASA concat

### CODE REVIEW

File: stimela/cargo/cab/calibrator/parameters.json [hunk 1]
```diff
@@ -33,6 +33,12 @@
             "dtype": "bool", 
             "default": false, 
             "name": "add-vis-model"
+        },
+        {
+            "info": "Visbility model column",
```
Comment (Athanaseus) [inline stimela/cargo/cab/calibrator/parameters.json]: s/Visbility/Visibility

File: stimela/cargo/cab/politsiyakat/parameters.json [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+            "name": "phase_range_clip"
+        }, 
+        {
+            "info": "Maximum fractional offset from median power in time and field before clipping", 
+            "dtype": "float", 
+            "default": "0.4", 
+            "name": "amp_frac_clip"
+        }, 
+        {
+            "info": "Maximum fraction of data flagged due to amplutude or phase accross calibrators before clipping baseline",
```
Comment (Athanaseus) [inline stimela/cargo/cab/politsiyakat/parameters.json]: s/amplutude/amplitude

Comment (bennahugo) [reply stimela/cargo/cab/politsiyakat/parameters.json]: Remove this cab. It is obsolete and replaced with other tasks.

**Comment by SpheMakh** [comment]
@bennahugo will remove on an upcoming PR

**Pull request merged**